extension/src/panel.ts

```diff
@@ -31,22 +32,20 @@ async function fileType(path: string): Promise<vscode.FileType | undefined> {
     const stat = await vscode.workspace.fs.stat(vscode.Uri.file(path))
     return stat.type
   } catch (err) {
-    // TODO: check exception and throw if unexpected
-    return undefined
+    throw new Error(`${err}`)
```

Comment:
You only want to throw on certain exceptions not when the path is not found for example

Reply:
Maybe instead of an error here we just want a message to be shown for when the path is not found? e.g.
```suggestion
        await vscode.window.showErrorMessage(`${path} not found`);
```

Not sure what other relevant errors there might be 🤔 An error is triggered elsewhere if the configured replayerEndpoint is a folder instead of a file.

Reply:
I've opted to remove this entirely and move the try/catch to `getReplayerEndpoint` since that's where we're handling the possible errors.

This follows a similar pattern to the example [here](https://github.com/microsoft/vscode-extension-samples/blob/0b3a31bf2bdd388ac4fdc0ccea2fb1315abfe3e3/fsconsumer-sample/src/extension.ts#L27-L32).